dist/s_version

```diff
@@ -18,42 +19,66 @@ while :
 	esac
 done
 
-# If the version hasn't changed and we're not forcing the issue, we're done.
-if test "$force" = no -a \
-    -f $m4dir/version.m4 -a \
-    -f $m4dir/version-set.m4 ; then
-	eval `grep '^VERSION_[A-Z]*=' $m4dir/version-set.m4`
-	if test x${WIREDTIGER_VERSION_MAJOR} = x${VERSION_MAJOR} -a \
-		x${WIREDTIGER_VERSION_MINOR} = x${VERSION_MINOR} -a \
-		x${WIREDTIGER_VERSION_PATCH} = x${VERSION_PATCH} ; then
-		exit 0
+GenAutoconfVersion(){
+	# If the version hasn't changed and we're not forcing the issue, we're done.
+	if test "$force" = no -a \
+		-f $m4dir/version.m4 -a \
+		-f $m4dir/version-set.m4 ; then
+		eval `grep '^VERSION_[A-Z]*=' $m4dir/version-set.m4`
+		if test x${WIREDTIGER_VERSION_MAJOR} = x${VERSION_MAJOR} -a \
+			x${WIREDTIGER_VERSION_MINOR} = x${VERSION_MINOR} -a \
+			x${WIREDTIGER_VERSION_PATCH} = x${VERSION_PATCH} ; then
+			return
```

Comment:
I realise we return instead of exitting here. Was this intended?

Reply:
Yep! We wanted to return instead of exiting, as we placed the two version gens (for autoconf and CMake) into functions so that we could split them and run both without exiting and stopping the other one wouldn't be affected and halted.